assets/js/modules/thank-with-google/components/setup/SetupPublicationActive.js

```diff
@@ -31,17 +31,25 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-import { MODULES_THANK_WITH_GOOGLE } from '../../datastore/constants';
 import Button from '../../../../components/Button';
+import useViewContext from '../../../../hooks/useViewContext';
+import { trackEvent } from '../../../../util';
+import { MODULES_THANK_WITH_GOOGLE } from '../../datastore/constants';
 import SetupPublicationScreen from './SetupPublicationScreen';
 const { useDispatch } = Data;
 
 export default function SetupPublicationActive( { currentPublicationID } ) {
 	const { setPublicationID } = useDispatch( MODULES_THANK_WITH_GOOGLE );
 
-	const handleSetupCustomize = useCallback( () => {
+	const viewContext = useViewContext();
+
+	const handleSetupCustomize = useCallback( async () => {
+		await trackEvent(
+			`${ viewContext }_thank-with-google`,
+			'review_publication_state'
```

Comment:
The event name should be `customize` here.
```suggestion
			'customize'
```